test/unit/process/test_cuda_process_set.cpp

```diff
@@ -99,17 +174,21 @@ using Group100000VectorMatrix = micm::VectorMatrix<T, 100000>;
 template<class T>
 using Group1000000VectorMatrix = micm::VectorMatrix<T, 1000000>;
 
-TEST(RandomProcessSet, Matrix)
+template<class T>
+using Group1000SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<1000>>;
+template<class T>
+using Group10000SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<10000>>;
+template<class T>
+using Group100000SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<100000>>;
+template<class T>
+using Group1000000SparseVectorMatrix = micm::SparseMatrix<T, micm::SparseMatrixVectorOrdering<1000000>>;
```

Comment:
If some of these aren't being used, I think you could remove them.

Reply:
removed.